doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -0,0 +1,100 @@
+[id='managing-business-central-configuring-githooks-proc']
+= Configuring git hooks
+
+{CENTRAL} can automatically push changes to a remote repository using Git hooks. After you configure the post-commit hook, {CENTRAL} automatically pushes your changes to the remote repository.
+
+[NOTE]
+====
+Currently only post-commit hooks are supported. Post-commit hooks are triggered after the commit process finishes.
+====
+
+.Procedure
+
+. Log in to {CENTRAL}.
+. Go to *Authoring* -> *Administration*.
+. Below the main menu, click *Repositories* -> *Clone repository*.
+. In the displayed Clone repository dialog box, fill in the repository information:
++
+* Repository Name
+* Organizational Unit
+* Git URL: For example, `https://github.com/USERNAME/REPOSITORY_NAME.git`
++
+[IMPORTANT]
+=====
+Use the HTTPS or Git protocol instead of a SCP-style SSH URL. {CENTRAL} does not support the basic SSH URL and an error appears if you use this URL.
+=====
++
+* User Name: your Git user name
+* Password: your Git password
+
+. In a terminal window, change to the created repository.
++
+[source]
+----
+cd JBOSS_HOME/bin/.niogit/REPOSITORY_NAME.git
+----
+
+. Change the remote URL:
```

Comment:
this is not needed anymore, the import preserves the origin

Reply:
Changed as suggested.